src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -135,13 +135,13 @@ operator()(const NodeID /*nid*/, const EdgeID source_edge_id, Intersection inter
     if (intersection_node_id == next->node)
         return intersection;
 
-    std::unordered_set<NameID> target_road_names;
+    std::vector<NameID> target_road_name_ids;
+    target_road_name_ids.reserve(next->intersection.size());
```

Comment:
Yeah. They are small. I just remember someone telling me to used `unordered_sets` for these ;)